### PR TITLE
MWPW-131002 - SKMS integration

### PR DESCRIPTION
**Pull request by amauch-adobe**
- Adds auto-approved CMR integration on Milo releases for SKMS.
- New GitHub action for merged pull requests against the production branch.

Resolves: [MWPW-131002](https://jira.corp.adobe.com/browse/MWPW-131002)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://skms-integration-final--milo--amauch-adobe.hlx.page/?martech=off

### CODE REVIEW

**Comment by amauch-adobe** [comment]
See [here](https://wiki.corp.adobe.com/pages/viewpage.action?pageId=3014303185) for instructions on how to test this integration.  Specifically, read my comment at the bottom in the "Comments".

**Comment by amauch-adobe** [comment]
Tested on my forked branch [here](https://github.com/amauch-adobe/milo/actions/runs/6580533901). See screenshots below from the SKMS system as well GitHub actions workflow run:

GitHub Action Job Result:
 
Screenshot 2023-10-19 at 3 52 34 PM

SKMS System CMR Created:

Screenshot 2023-10-19 at 3 44 42 PM

**Comment by amauch-adobe** [comment]
The failure in the Unit Tests has to do with the FEDS Gnav test, which this commit wouldn't have affected in any way, shape, or form.

Screenshot 2023-11-14 at 6 02 33 PM

File: .github/workflows/skms.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            local method=$4
+            local method_params=$5
+            local url=$6
+
+            if [ -z "$username" ]; then
+              echo "First parameter missing, must be SKMS username"
+              return 1
+            fi
+
+            if [ -z "$passkey" ]; then
```
Comment (mokimo) [inline .github/workflows/skms.yaml:90]: Just to be sure, passkey is not a secret - correct? Since we log this and this is public.

**Review by mokimo — APPROVED** [review]

File: .github/workflows/skms.yaml (hunk 1 above)
Comment (amauch-adobe) [reply .github/workflows/skms.yaml:90]: @mokimo, excellent question, it is a secret that is coming in from a GitHub repository secret variable so it is masked as '*' in the logs as shown below in the GitHub Actions Workflow printout:

Screenshot 2023-12-05 at 9 49 29 AM

and

Screenshot 2023-12-05 at 9 49 39 AM

So we should be good.

**Pull request merged**